pinot-query-runtime/src/main/java/org/apache/pinot/query/runtime/executor/OpChainSchedulerService.java

```diff
@@ -78,29 +78,35 @@ protected void run()
         }
 
         OpChain operatorChain = _scheduler.next();
+        LOGGER.trace("({}): Scheduling", operatorChain);
         _workerPool.submit(new TraceRunnable() {
           @Override
           public void runJob() {
             try {
-              ThreadResourceUsageProvider timer = operatorChain.getAndStartTimer();
+              LOGGER.trace("({}): Executing", operatorChain);
+              operatorChain.getStats().executing();
 
               // so long as there's work to be done, keep getting the next block
               // when the operator chain returns a NOOP block, then yield the execution
               // of this to another worker
               TransferableBlock result = operatorChain.getRoot().nextBlock();
               while (!result.isNoOpBlock() && !result.isEndOfStreamBlock()) {
-                LOGGER.debug("Got block with {} rows.", result.getNumRows());
```

Comment:
I removed this because it always returns `0` - `MailboxSendOperator` either returns `No-op` or `EOS`